test/test_helper.rb

```diff
@@ -89,9 +89,20 @@ def after_teardown
   end
 end
 
+module TestCaseNetworkStubExtensions
+  def setup
```

Comment:
What about `Resolv.getaddress` and `Resolv.getname`? Or are they covered because they just instantiate a `Resolv::DNS` instance?

Reply:
Yep, that's how I understand the code however I haven't tested. Let me check.